src/Utils/request/errorHandler.ts

```diff
@@ -32,11 +32,19 @@ export function handleHttpError(error: Error) {
 
   if (isBadRequest(error)) {
     const errs = cause?.errors;
-    if (isPydanticError(errs)) {
+    if (errs && isPydanticError(errs)) {
```

Comment:
isPydanticError will anyways return false if errs is false right?

```suggestion
    if (isPydanticError(errs)) {
```